test-it/testList.json

```diff
@@ -31,7 +31,8 @@
         },
         "manual": {
             "tests": [
-                "StringFormatTime.via"
+                "StringFormatTime.via",
+                "TestVI_LargeBDTest.via"
```

Comment:
I just created a defect to look into the issue later on: DE14000 Vireo: Investigate why js test framework fails when TestVI_LargeBDTest.via is included.